pom.xml

```diff
@@ -5,12 +5,146 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.example</groupId>
-    <artifactId>test</artifactId>
+    <artifactId>morozova</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <dependencies>
+
+        <!-- https://mvnrepository.com/artifact/org.testng/testng -->
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.3.0</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>junit</groupId>
+                    <artifactId>junit</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/io.github.bonigarcia/webdrivermanager -->
+        <dependency>
+            <groupId>io.github.bonigarcia</groupId>
+            <artifactId>webdrivermanager</artifactId>
+            <version>4.4.1</version>
+        </dependency>
+
+        <!--<dependency>
```

Comment:
looks like from branch Homework_2 calculator can not be seen, because it is on branch Homework_1:
 'dependencies.dependency.systemPath' for calculator-1.0:calculator-1.0.:jar refers to a non-existing file C:\Users\user\IdeaProjects\test\src\lib\calculator-1.0.jar @ line 38, column 25.
So I commented it out